### ISSUE TITLE
training speed when adding DISTS loss

### ISSUE THREAD

**Issue by zhengbowei** [dingkeyan93/IQA-optimization#2]
Dear author:
I add DISTS loss to my task but the model training speed is very slow, Is it normal?
code:
def forward():
   device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
   D = DISTS().to(device)
   dists_loss = D(X, Y, as_loss=True)

**Comment by dingkeyan93** [comment]
Yes, because of VGG, the speed of DISTS is slow compared with SSIM.
